src/ucp/wireup/address.c

```diff
@@ -1434,21 +1434,18 @@ ucs_status_t ucp_address_length(ucp_worker_h worker,
     ucp_address_packed_device_t *devices;
     ucp_rsc_index_t num_devices;
     ucs_status_t status;
-    size_t size;
 
     /* Collect all devices required to pack their address */
-    status = ucp_address_gather_devices(worker, ep_config, tl_bitmap,
-                                        pack_flags, addr_version,
-                                        &devices, &num_devices);
+    status = ucp_address_gather_devices(worker, key, tl_bitmap,
+            pack_flags, addr_version, &devices, &num_devices);
```

Comment:
Done. I'll try to align the ```(``` and the next head character at same column at most places. If it can't do that, I'll indent the next line little to not exceed 80 characters at one line.